src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -88,15 +88,26 @@ export const TradeConfirm = ({ history }: RouterProps) => {
       // Poll until we have a "buy" txid
       // This means the trade is just about finished
       const txs = await poll({
-        fn: () => getTradeTxs(result),
-        validate: (txs: TradeTxs) => !!txs.buyTxid,
+        fn: async () => {
+          try {
+            return { ...(await getTradeTxs(result)) }
+          } catch (e) {
+            return { sellTxid: '', buyTxid: '', e }
+          }
+        },
+        validate: (txs: TradeTxs & { e: Error }) => !!txs.buyTxid || !!txs.e,
         interval: 10000, // 10 seconds
         maxAttempts: 300, // Lots of attempts because some trade are slow (thorchain to bitcoin)
       })
-      if (!txs.buyTxid) throw new Error('No buyTxid from getTradeTxs')
+
+      if (txs.e) throw txs.e
+      if (!txs?.buyTxid) throw new Error('No buyTxid from getTradeTxs')
```

Comment:
translations copy for this error message?

Reply:
unless fallback to general error is ok, not exactly sure how error toast works

Reply:
maybe could use a translation but its not related to this pr

Reply:
should at least ask product and make an issue based on that outcome or it will never be addressed

Reply:
Ya there are probably quite a few throws we could translate if we feel its necessary

Reply:
We will need to tackle this in a consistent way as we don't currently use translation strings for errors across the codebase. Agree this belongs in a whole separate issue and we should create it